docs/setup-module.md

```diff
@@ -0,0 +1,55 @@
+# Setting up a lean custom module in Rust
+1. Currently `cb_*` crates are not uploaded to crates.io , so easiest way to get started is to clone the commit-boost-client repo and initialize a new module in the examples folder
+2. Initialize a new project using `cargo new <my_module_name>`
+3. The Cargo.toml needs to have the cb_* dependencies from the root project
+```[dependencies]
+cb-cli.workspace = true
+cb-common.workspace = true
+cb-pbs.workspace = true
+cb-crypto.workspace = true```
+4. Import the cb modules in your main.rs:
+```use cb_common::{
```

Comment:
this is not formatted properly.

Make sure that it is rendered properly and you can include the ```rust tag so that there is syntax highlights